hashtagsv2/hashtags/templates/hashtags/index.html

```diff
@@ -29,10 +29,14 @@
                 {{ form.startdate.label }}
                 {{ form.startdate }}
               </div>
-              <div class="three columns">
+              <div class="three columns" style="margin-right: 2em">
```

Comment:
This looks good, and in-line, but is probably somewhat confusing to users. We don't even explain that multiple hashtags are supported in the search bar, so seeing an option for and/or isn't likely to make much sense. I'm not sure what the best solution is.

Perhaps we could have an Advanced Search menu that's hidden by default via javascript. Could you try that?

Reply:
```suggestion
              <div class="three columns" style="margin-right: 2em">
```
Use percentages rather that a fixed value so users dont face problems in using it .
For centering the ```div```, use this css style -
<div class="three columns" style="transform: translateX(-50%); left: 50%; position: absolute;">

Reply:
Use this css for better view. It automatically collapses in mobile and tablet viewports.
``
<div class="row">
        <div class="col-sm-4">
         <!-- col 1 here -->
          <div class="col-sm-4">
         <!-- col 2 here -->
          <div class="col-sm-4">
         <!-- col 3 here -->
</div>
``

Reply:
Use this css for better view. It automatically collapses in mobile and tablet viewports.

```
<div class="row">
        <div class="col-sm-4">
         <!-- col 1 here -->
          <div class="col-sm-4">
         <!-- col 2 here -->
          <div class="col-sm-4">
         <!-- col 3 here -->
</div>
```

Reply:
```Perhaps we could have an Advanced Search menu that's hidden by default via javascript. Could you try that?```
Javascript is not needed.
See this -
```
<style>
#stuffToShow {
    display:none;
}

#checkbox {
    opacity:0.01;
}

#checkbox:checked + #stuffToShow {
    display:block;
}
</style>

<label for="checkbox" id="checkboxLabel">Show More</label>
<input type="checkbox" id="checkbox" />
<div id="stuffToShow">
    Happy times
</div>
```